docs/sdk/sdk-by-examples/simple-governance/submit-proposal.md

```diff
@@ -6,6 +6,23 @@ Uuse the CLI to create a new proposal:
 simplegovcli propose --title="Voting Period update" --description="Should we change the proposal voting period to 3 weeks?" --deposit=300Atoms
 ```
 
+Or, via a json file:
+
+```bash
+simplegovcli propose --proposal="test.json"
```

Comment:
I suggest changing this to: `$ simplegovcli propose --proposal=path/to/proposal.json`